src/codeu/chat/database/Packer.java

```diff
@@ -0,0 +1,59 @@
+package codeu.chat.database;
+import codeu.chat.util.Time;
+import codeu.chat.util.Uuid;
+import codeu.chat.common.Conversation;
+import codeu.chat.common.ConversationSummary;
+import codeu.chat.common.Message;
+import codeu.chat.common.User;
+import org.bson.Document;
+
+import java.io.IOException;
+
+public class Packer {
+
+    /**
+     * Converts a User object to a Document
+     * @param user: the User to be converted
+     * @return a Document derived from Message
+     */
+    public static Document packUser(User user) {
+        System.out.println("Packing UserID: " + user.id.toString());
```

Comment:
it would be nice to use logging consistently instead of sometimes logging and sometimes direct println. Or, if there are reasons supporting it, go consistently the other direction.